can/tests/test_define.py

```diff
@@ -1,35 +1,27 @@
-#!/usr/bin/env python3
-import unittest
-
 from opendbc.can.can_define import CANDefine
 from opendbc.can.tests import ALL_DBCS
 
 
-class TestCADNDefine(unittest.TestCase):
+class TestCADNDefine:
   def test_civic(self):
 
     dbc_file = "honda_civic_touring_2016_can_generated"
     defs = CANDefine(dbc_file)
 
-    self.assertDictEqual(defs.dv[399], defs.dv['STEER_STATUS'])
-    self.assertDictEqual(defs.dv[399],
-                         {'STEER_STATUS':
+    assert defs.dv[399] == defs.dv['STEER_STATUS']
+    assert defs.dv[399] == \
+                         {'STEER_STATUS': \
```

Comment:
```suggestion
                         {'STEER_STATUS':
```

I don't think you need backslash here?